src/core/BanksUser.ts

```diff
@@ -66,6 +71,28 @@ export class BanksUser {
       method: 'GET',
     });
 
-    return new BanksUser(banksUser);
+    return new BanksUser(banksUser, requestBuilder);
+  }
+
+  /**
+   * Update a banksUser
+   * @param body Patch banks user request body
+   */
+  public async update(body: Partial<IBanksUser>): Promise<void> {
```

Comment:
Could you please create a new `PatchBanksUserDTO` interface in the `src/lib/Algoan.dto.ts` file?

Reply:
Yeah! Done :)